framework/wazuh/core/cluster/master.py

```diff
@@ -716,11 +720,13 @@ async def sync_integrity(self, task_id: str, received_file: asyncio.Event):
                     raise exc_info
 
                 # Send zip file to the worker into chunks.
-                await self.send_file(compressed_data, task_id)
+                time_to_send = time()
+                sent_size = await self.send_file(compressed_data, task_id)
+                time_to_send = time() - time_to_send
```

Comment:
It would be advisable to use [`perf_counter`](https://docs.python.org/3/library/time.html#time.perf_counter) here instead of time, as it returns the value of a performance counter with the highest available resolution to measure a short duration. We agreed on using it for this kind of situation.